CODEOWNERS

```diff
@@ -0,0 +1,13 @@
+# This file helps Github doing automatic review requests for new PRs.
+# It should always list the active maintainers of certain add-ons.
+
+# As a fallback, if no specific maintainer is listed below, assign the PR to the repo maintainers team:
+*       @openhab/2-x-add-ons-maintainers
+
+# Add-on maintainers:
+/addons/binding/org.openhab.binding.deconz/ @davidgraeff
+/addons/binding/org.openhab.binding.enocean/ @fruggy83
```

Comment:
According to the [docs](https://help.github.com/articles/about-code-owners/) this might not work:

>  The people you choose as code owners must have write permissions for the repository.

Reply:
Let's try what happens. But it should be fine to also give write access to all add-on maintainers as well (as we can still setup a "require review by repo-maintainer"), but I'd then first want to clearly agree & communicate the rules that this would bring with it.

Reply:
I can only imagine what might happen if it does work and people branch this repo. Then you'll be spammed with reviews whenever someone creates a PR on a fork with your name in this file. :wink:

Reply:
Yes I agree we can see how this works first.